### PR TITLE
Add Right Margin Guideline

### PR DESCRIPTION
**Pull request by tbpassin**
Add optional right margin guideline in body. A new help menu item is also added to explain the settings.  Tested with both pyqt5 and pyqt6.

test-all:
```
Ran 923 tests in 20.029s
OK (skipped=8)
```

pylint: 9.94/10

### CODE REVIEW

**Review by edreamleo — CHANGES_REQUESTED** [review]
Please create the help menu entries by adding entries to leoSettings.leo. See:

leoSettings.leo#@settings-->Menus-->@menus-->@menu &Help-->@menu &Open Help topics

**Comment by tbpassin** [comment]
OK, I'll get at that.

**Comment by tbpassin** [comment]
Done - help menu items now defined in leoSettings.leo.

**Review by edreamleo — COMMENTED** [review]
The code looks good. Not sure about whether the help text is in the right place, but that's a nit that we can handle after the merge.

I won't approve the PR yet, because it should be merged into devel after releasing 6.6.1. I'll approve when that happens.

**Comment by tbpassin** [comment]
"All completed 6.6.2 work has been merged into devel".  Not quite : )

**Comment by edreamleo** [comment]
@tbpassin Thanks for the reminder. I have just merged this PR.

**Comment by tbpassin** [comment]
Thanks, Edward!

**Pull request merged**
